packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/nodejs/integrationsteps.md

```diff
@@ -1,121 +1,122 @@
-> **Note:** These steps describe integrating the Okta email OTP flow into your app. To learn more about Okta email including how to integrate Okta email using magic links, see the [Okta email (magic link/OTP) integration guide](/docs/guides/authenticators-okta-email/nodeexpress/main/).
+### 1. Select the password recovery link
```

Comment:
This is what the user will do , not what the developer needs to do.  You seem to have re-written this step to target the user, not the developer. I'd suggest reverting this text back to "Add a forgot password link" etc. and the rest of the text

Reply:
Let's sync up about this one tomorrow.

Reply:
Updated based on our conversations.